app/timetables/models.py

```diff
@@ -78,6 +78,7 @@ class Timetable(TimestampMixin):
     current_cycle_day = models.PositiveSmallIntegerField(
         validators=[MinValueValidator(1)]
     )
+    description = models.TextField(blank=True)
```

Comment:
@andela-joyebanji: I understand this is an hassle, but this modification should not be part of this issue/story.